### PR TITLE
[BugFix] The db_admin role miss some privilege type in new RBAC

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 39 / 39 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/privilege/PrivilegeManager.java|39|39|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java [hunk 1]
```diff
@@ -277,16 +279,12 @@ private void initPrivilegeCollectionAllObjects(
...
-            RolePrivilegeCollection collection = new RolePrivilegeCollection(
-                    name, RolePrivilegeCollection.RoleFlags.MUTABLE);
-            roleIdToPrivilegeCollection.put(roleId, collection);
-            roleNameToId.put(name, roleId);
-            LOG.info("create built-in role {}[{}]", name, roleId);
-            return collection;
-        }
-        // public roles may be changed and persisted in image before restarted
-        return null;
+        RolePrivilegeCollection collection = new RolePrivilegeCollection(
```
Comment (nshangyiming) [inline fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java:282]: Always reinitiate the built-in role won't affect correctness, but i think the better way is to keep this fucntion and 
related condition unchanged, and load the info of the built-in role from image.

File: fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java [hunk 2]
```diff
@@ -277,16 +279,12 @@ private void initPrivilegeCollectionAllObjects(
...
+                name, RolePrivilegeCollection.RoleFlags.MUTABLE);
+        roleIdToPrivilegeCollection.put(roleId, collection);
+        roleNameToId.put(name, roleId);
+        LOG.info("create built-in role {}[{}]", name, roleId);
+        return collection;
```
Comment (nshangyiming) [inline fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeManager.java:287]: And could you write some details about this bug in the pr description and add some UTs?

**Review by nshangyiming — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 39 / 39 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/privilege/PrivilegeManager.java|39|39|100.00%|[]|

**Comment by HangyuanLiu** [comment]
run DCO

**Comment by HangyuanLiu** [comment]
run starrocks_admit_test

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Review by gengjun-git — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**